### PR TITLE
feat: update chart

### PR DESCRIPTION
**Pull request by viceice**
Moved renovate chart on artifacthub.io to new org https://artifacthub.io/packages/helm/renovate/renovate

Maybe we can /should add some metadata? (eg using `artifacthub.io/links` to nicely name links)
https://artifacthub.io/docs/topics/annotations/helm/#supported-annotations

### CODE REVIEW

**Comment by JamieMagee** [comment]
Is this ready for review?

File: charts/renovate/Chart.yaml [hunk 1]
```diff
@@ -13,5 +13,15 @@ keywords:
   - update
 sources:
   - https://github.com/renovatebot/renovate
+  - https://github.com/renovatebot/helm-charts
 maintainers:
   - name: JamieMagee
```
Comment (viceice) [inline charts/renovate/Chart.yaml:18]: Yes, should i add your email too?

**Comment by viceice** [comment]
@JamieMagee Ready for review now. 😉

**Comment by viceice** [comment]
@rarkins Should I add you as maintainer too?

File: charts/renovate/Chart.yaml (hunk 1 above)
Comment (JamieMagee) [reply charts/renovate/Chart.yaml:18]: Sure, why not. `jamie.magee@gmail.com`

**Review by JamieMagee — APPROVED** [review]

**Pull request merged**